.templates/base_script.sh.template

```diff
@@ -0,0 +1,95 @@
+{% block header %}
+#!/usr/bin/env bash
+
+# Automatically generated by nengo-bones, do not edit this file directly
+
+NAME=$0
+COMMAND=$1
+STATUS=0  # used to exit with non-zero status if any command fails
+
+exe() {
+    echo "\$ $*";
+    args=( "$@" )
+    # remove empty spaces from args
+    for i in "${!args[@]}"; do
+      [ -n "${args[$i]}" ] || unset "args[$i]"
+    done
+    # put quotes around all arguments except the first
+    for i in "${!args[@]}"; do
+      if [[ $i -gt 0 ]]; then
+        args[$i]='"'${args[$i]}'"'
+      fi
+    done
+    eval "${args[@]}" || { echo -e "\033[1;31mCOMMAND '${args[0]}' FAILED\033[0m"; STATUS=1; }
```

Comment:
Looks like this part is the only change from the current base script, just for my own interest why was this required?